docs/prometheus-metrics.md

```diff
@@ -0,0 +1,22 @@
+# Prometheus Metrics
+
+Both Events and Configuration API exposes Prometheus metrics. The metrics are accesible via `/v1/metrics` endpoint.
+
```

Comment:
For future, writing an prometheus [exporter](https://prometheus.io/docs/instrumenting/exporters/) for EG metrics might be really useful for those who want to integrate with EG.

Reply:
Is it possible to add some screenshots of the charts that we have been doing via Grafana and EG data?

Reply:
The next step is to add Grafana dashboard JSON file. I will do that In separate PR.

Reply:
👍